akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -148,6 +148,20 @@ akka.cluster.sharding {
   # be the same as "akka.cluster.sharding.role".
   # A lease can be configured in these settings for the coordinator singleton
   coordinator-singleton = ${akka.cluster.singleton}
+
+  coordinator-state {
+    # State updates are required to be written to a majority of nodes plus this
+    # number of additional nodes. Can also be set to "all" to require
+    # writes to all nodes. The reason for write/read to more than majority
+    # is to have more tolerance for membership changes between write and read.
```

Comment:
Should we mention also the trade off with writes being slower?

Reply:
added